app/src/components/realmAdminSvc.js

```diff
@@ -231,6 +231,26 @@ class RealmAdminService {
     return response.data;
   }
 
+  async generateNewClientSecret(id) {
+    if (!id) {
+      log.error('RealmAdminService generateNewClientSecret id parameter is null.');
```

Comment:
log.error('RealmAdminService.generateNewClientSecret', 'id parameter is null.');

Reply:
Fixed